### ISSUE TITLE
Migration database from v1.2.7-beta to v4.6.3 - build 3866

### ISSUE THREAD

**Issue by goku86fra** [snipe/snipe-it#6074]
Dear All,

I am trying to migrate the database of my asset from v1.2.7-beta to v4.6.3 - build 3866, but I have some problems. I tried to create a backup on phpmyadmin and to import it on the new software, but there are some tables that don't have the correspondence. Could you please help me to proceed? I cannot find the documentation for my old version.

Many thanks!

**Comment by EarlRamirez** [comment]
@goku86fra, wow! I think you will have to do increment upgrades to get to v4.6.3.

**Comment by goku86fra** [comment]
@EarlRamirez , I followed your feedback starting to install the v2, but during the installation of composer:
"composer install --no-dev --prefer-source" I obtine the v4.6 directly. With the same issues beetwen different database.
Any idea how i can proceed?

**Comment by snipe** [comment]
Please provide specific error messages that you're getting.

**Comment by goku86fra** [comment]
Hi, thanks all for the feedback.
I managed to do the migration :) with phpmyadmin, doing all Table to Table with the copy in the Tab "Operation". 
Now I have all assets, accessories and users also on the new, with all the status label, but I cannot see the check out to the user... it is very strange, the assets are in the correct status label, but if I do a search in Checked out to I cannot see the name of the users. Does someone have an idea to solve it?

Many thanks!!!

**Comment by goku86fra** [comment]
Hi everybody,

Just now snipe-it works more or less fine 
So now after to adapt the old asset to my new asset the problem is the next:
All document charged for every asset has been lost...
I recover all this document from the older server (Linux server) allocated in /var/www/html/app/private_uploads
Also, I checked that in my database after the importation on table "asset_logs" there is the table "filename"
which point to this file.

So now on my new server (server windows with WAMP), I tried to leave all this file on this path:

www/snipe-it/storage/private_uploads/asset

but the files are not showed on the application via web.
I charge some files on my new asset and verified that these files are saved on: storage/private_uploads/asset
but the old document no.

Is there some way to make that the old files are inserted again on my new asset?

Many thanks!!!
